sda-svc/values.yaml

```
@@ -44,11 +44,11 @@ global:
 # There are a number of network providers that support NetworkPolicy, including:
 # Calico, Cilium, Kube-router, Romana and Weave Net
   networkPolicy:
-    create: false
-    brokerNamespace: ""
-    databaseNamespace: ""
-    externalNamespace: ""
-    internalNamespace: ""
+    create: true
```

this can be set during install, let us not default it to `true`
```suggestion
    create: false
```